vertical-pod-autoscaler/e2e/go.mod

```diff
@@ -85,115 +103,114 @@ require (
 	github.com/modern-go/reflect2 v1.0.3-0.20250322232337-35a7c28c31ee // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
 	github.com/mxk/go-flowrate v0.0.0-20140419014527-cca7078d478f // indirect
-	github.com/opencontainers/cgroups v0.0.4 // indirect
+	github.com/opencontainers/cgroups v0.0.5 // indirect
```

Comment:
https://github.com/kubernetes/kubernetes/pull/135504

It seems we can't upgrade to 0.0.6 or higher until the above PR is merged and we depend on that version of kubernetes